src/pybind/mgr/dashboard/controllers/smb.py

```diff
@@ -87,6 +88,19 @@
     }, "Configuration for the CephFS share")
 }
 
+def raise_unsuccess(func):
```

Comment:
grammar nitpick: `raise_unsucess` is awkward, unsuccess isn't typical, the opposite of "success" is "failure": try something like `raise_on_failure` `raise_if_failure`, `catch_failure`, or the like. If you dislike these suggestions, just ask and I can provide more.

Reply:
You are right, it is a wierd wording, changed it to: `raise_on_failure`